.github/workflows/release.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
         with:
           context: .
           push: true
-          build-args: VERSION=105
+          build-args: VERSION=Brave-1.0
```

Comment:
Hm, this should probably just use the VERSION computed previously, based on the current tag?